packages/big-design/src/components/Panel/Contents/Contents.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { memo, ReactNode } from 'react';
+
+import { BoxProps } from '../../Box';
+
+import { StyledPanelContents, StyledPanelContentsWrapper } from './styled';
+
+export interface PanelContentProps extends BoxProps {
+  children?: ReactNode;
+  padded?: boolean;
+  height?: string;
+  scrollable?: boolean;
```

Comment:
would we ever want to support a scenario where the height is fixed, but scrolling is disabled?

would this just result in content being abruptly cropped and unreachable?